config/zones/CL-SEN.yaml

```diff
@@ -257,5 +263,5 @@ parsers:
   productionCapacity: CL_SEN.fetch_production_capacity
 sources:
   Electricity Maps, CEN:
-    link: https://docs.google.com/spreadsheets/d/1-iXUGhAM_9Ft5ejlNVxaqaAwG8Octh3gSjU-dm4e96o/edit#gid=0
+    link: https://docs.google.com/spreadsheets/d/1-iXUGhAM_9Ft5ejlNVxaqaAwG8Octh3gSjU-dm4e96o/edit#gid=163148677 (average thermoelectric emission factors)
```

Comment:
The spreadsheet is not publicly accessible, is this intentional?

Reply:
Try now, I updated the permissions. I believe all our sources are intended to be public.

Reply:
Fixed, thank you!